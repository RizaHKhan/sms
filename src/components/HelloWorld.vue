<template>
  <div class="hello">
    <h1>{{ message }}</h1>
  </div>
</template>

<script>
export default {
  name: 'HelloWorld',
  data: function () {
    return {
      message: 'Hello World'
    };
  }
};
</script>
