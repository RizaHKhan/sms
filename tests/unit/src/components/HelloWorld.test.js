import { shallowMount } from '@vue/test-utils';
import HelloWorld from '@/components/HelloWorld.vue';

describe('HelloWorld.vue', () => {
  test('Renders correctly', () => {
    const wrapper = shallowMount(HelloWorld);

    expect(wrapper)
      .toMatchSnapshot();
  });
});
