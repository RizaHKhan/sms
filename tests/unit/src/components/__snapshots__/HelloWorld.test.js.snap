// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld.vue Renders correctly 1`] = `
<div class="hello">
  <h1>Hello World</h1>
</div>
`;
